modules/openapi-generator/src/main/resources/kotlin-client/libraries/multiplatform/infrastructure/ApiClient.kt.mustache

```diff
@@ -60,16 +51,9 @@ import {{packageName}}.auth.*
     {{/hasAuthMethods}}
 
     {{#nonPublicApi}}internal {{/nonPublicApi}}companion object {
+        const val BASE_URL = "{{{basePath}}}"
+        const val JSON_DEFAULT = Json { ignoreUnknownKeys = true }
```

Comment:
```
Const 'val' has type 'Json'. Only primitives and String are allowed 
```
Some tests that would make CI catch it would be really good to have :sweat_smile:

Reply:
Oh man, obviously 🙄 Well spotted